components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/handlers/TokenResponseTypeHandler.java

```diff
@@ -565,4 +562,16 @@ private boolean isNotRenewAccessTokenPerRequest() {
         }
         return !isRenew;
     }
+
+    private static AccessTokenDO getAccessTokenCheckingContext(String accessToken,
+                                                               OAuthAuthzReqMessageContext msgCtx) throws IdentityOAuth2Exception {
+
+        Object accessTokenObject = msgCtx.getProperty(OAuth2Util.ACCESS_TOKEN_DO);
+        if (accessTokenObject != null && accessTokenObject instanceof AccessTokenDO) {
```

Comment:
Here null check on `accessTokenObject` is redundant since `instanceof` would return false if its first operand is null

Reply:
Ok, I'll remove the reduandancy